scripts/package-build/blackbox_exporter/package.toml

```diff
@@ -1,5 +1,5 @@
 [[packages]]
 name = "blackbox_exporter"
-commit_id = "v0.25.0"
+commit_id = "v0.26.0"
```

Comment:
Is there a reason or specific bug to update this?

Reply:
nothing major, a few improvements and small bugfixes.